src/DmxInput.cpp

```diff
@@ -64,12 +73,41 @@ DmxInput::return_code DmxInput::begin(uint pin, uint start_channel, uint num_cha
     _start_channel = start_channel;
     _end_channel = start_channel + num_channels;
 
+    _dma_chan = dma_claim_unused_channel(true);
+
+   //initialize with some starting values
+    for(int i=0;i<_buffer_size();i++) {
+        ((volatile uint8_t*)_buf)[i] = i;
```

Comment:
Would it be possible to use `memset` here instead?

Reply:
Not with the increasing `i`, right?
I added this code mostly for debugging. For production there isn't actually a reason to initialize the memory at all.